### DISCUSSION TITLE
Add custom layout `VariableSizedWrapGrid`?

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#572] [New Feature Discussions]
@jsuarezruiz Added to his .NET MAUI Good Looking challenge a layout called `VariableSizedWrapGrid`.  See [VariableSizedWrapGrid.cs](https://github.com/jsuarezruiz/netmaui-beautyshop-app-challenge/blob/main/src/NET%20MAUI/BeautyShopApp/Controls/VariableSizedWrapGrid.cs).

Could we look to bring this into MCT, as I think this would appeal to lots of app styles?  Javier has done the vast majority of the work making this .NET MAUI ready, but would need integrating, samples and unit tests.

I did ask if he was OK with getting this added to MCT [See tweet](https://twitter.com/GeorgeLeithead/status/1562819584356663297?s=20&t=Zgm6B_IJ6isok1H68csS-Q)

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
